src/channels.ts

```diff
@@ -158,9 +158,8 @@ export default class Channels {
       }
       const channels = await response.json()
```

Comment:
```suggestion
      const channelsPage: ChannelsPage = await response.json()
      return channelsPage
```

There is now no need for the deconstruction below. Fix for the rest